frontend/__mocks__/operatorHubItemsMocks.ts

```diff
@@ -1,6 +1,7 @@
 /* eslint-disable no-unused-vars, no-undef */
 
 import { PackageManifestKind } from '../public/components/operator-lifecycle-manager';
+import { OperatorHubItem } from '../public/components/operator-hub';
```

Comment:
Updated the mocks here so they match the `OperatorHubItem` type that is passed to the tile components.